.azure/pipelines/examples-test.yml

```
@@ -127,19 +130,18 @@ jobs:
         displayName: Setup Infracost
         inputs:
           apiKey: $(infracostApiKey)
-      - bash: >
-          infracost output --path="infracost_jsons/*.json" --format=json
-          --out-file=/tmp/infracost_combined.json
-        displayName: Combine the results
-      - task: InfracostComment@0
-        displayName: Post the comment
-        inputs:
-          githubToken: $(githubToken)
-          path: /tmp/infracost_combined.json
-          behavior: update
-          dryRun: true
+      - bash: |-
+          infracost comment github \
+          --path infracost_jsons/*.json \
```

```suggestion
          --path "infracost_jsons/*.json" \
```

This causes the failed matrix test.